bin/vue-init

```diff
@@ -62,9 +63,11 @@ var to = path.resolve(rawName || '.')
 var clone = program.clone || false
 
 var tmp = path.join(home, '.vue-templates', template.replace(/\//g, '-'))
-if (program.offline) {
+if (program.offline && exists(tmp)) {
   console.log(`> Use cached template at ${chalk.yellow(tildify(tmp))}`)
   template = tmp
+} else {
+  rm(tmp)
```

Comment:
Native `/tmp` (😝  And whatever alternative for windows) directory can be used for that.